File: src/main/java/com/ort/app/web/form/VillageActionForm.java
package com.ort.app.web.form;

import java.io.Serializable;

import javax.validation.constraints.NotNull;

public class VillageActionForm implements Serializable {

    private static final long serialVersionUID = 1L;

    @NotNull
    private String myself;

    private String target;

    /** 発言 */
    @NotNull
    private String message;

    public String getMyself() {
        return myself;
    }

    public void setMyself(String myself) {
        this.myself = myself;
    }

    public String getTarget() {
        return target;
    }

    public void setTarget(String target) {
        this.target = target;
    }

    public String getMessage() {
        return message;
    }

    public void setMessage(String message) {
        this.message = message;
    }
}

File: src/main/kotlin/com/ort/app/api/request/VillageActionForm.kt
package com.ort.app.api.request

import jakarta.validation.constraints.NotNull

data class VillageActionForm(
    @field:NotNull
    val myself: String? = null,

    val target: String? = null,

    @field:NotNull
    var message: String? = null,

    val convertDisable: Boolean? = null
)